<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComputerWindow</class>
 <widget class="QMainWindow" name="AddComputerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textNewNameC">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textNewBuildYear">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textNewType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>53</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Build Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>266</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Was it built?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkYes">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkNo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddC">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelC_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabelC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>326</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_computer">
    <property name="title">
     <string>Add computer</string>
    </property>
   </widget>
   <addaction name="menuAdd_computer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
